test/scripts/console/deploy.js

```diff
@@ -77,7 +77,7 @@ describe('deploy', () => {
     });
   });
 
-  it('deployer not found');
+  // it('deployer not found');
```

Comment:
Instead of using `x pending` to show missing test, I suggest to add a note, like

```
// it('deployer not found'); missing-unit-test
```

Then we can mention it in the [roadmap](https://github.com/hexojs/hexo/issues/2492), and contributor can simply search for "missing-unit-test" to locate missing tests.